robot/Cumulus/resources/NPSPSettingsPageObject.py

```diff
@@ -146,5 +146,17 @@ def verify_gift_entry_is_not_enabled(self):
                 self.click_toggle_button("Gift Entry")
                 self.wait_for_message("Gift Entry Disabled")
             else:
-                self.builtin.log("As expected Gift Entry is Disabled")    
-                                          
\ No newline at end of file
+                self.builtin.log("As expected Gift Entry is Disabled")
+
+    def Check_Crlp_Not_Enabled_By_Default(self):
+        """Verifies that advanced mapping is not enabled by default
```

Comment:
you forgot to change this to customizable rollups

Reply:
Thanks. Updated